webpack.config.js

```diff
@@ -1,114 +1,111 @@
 const path = require('path');
-const MinifyPlugin = require('babel-minify-webpack-plugin');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
 
-const env = process.env.NODE_ENV || 'dev';
+// we build the library for two different build targets:
+// static (with scoped styles) and themeable (with stock,
+// overrideable classnames)
+const targetProperties = [{
```

Comment:
I know I could have written this as a one-liner, but I split it out into its own variable for clarity's sake.